ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/AStarFootstepPlanner.java

```diff
@@ -202,12 +202,15 @@ public void handleRequest(FootstepPlannerRequest request, FootstepPlannerOutput
          reportStatus(request, outputToPack);
          return;
       }
-
-      // Start planning loop
+      else
```

Comment:
Ok so we never set the goal pose that gets logged. In several logs I looked at this getting logged:
```
    "goal_pose": {
      "position": {
        "x": "NaN",
        "y": "NaN",
        "z": "NaN"
      },
      "orientation": {
        "x": "NaN",
        "y": "NaN",
        "z": "NaN",
        "w": "NaN"
      }
    },
```